api-tester/build.gradle

```diff
@@ -1,4 +1,37 @@
-apply from: "$rootProject.projectDir/library.gradle"
+apply plugin: 'com.android.library'
+apply plugin: 'kotlin-android'
+apply plugin: 'kotlin-android-extensions'
+
+android {
+    compileSdkVersion compileVersion
+    buildToolsVersion buildToolsVersion
+
+    defaultConfig {
+        minSdkVersion minVersion
+        targetSdkVersion compileVersion
+
+        testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
+        consumerProguardFiles "consumer-rules.pro"
+    }
+    sourceSets {
+        main.java.srcDirs += 'src/main/kotlin'
+        main.java.srcDirs += 'src/main/java'
+        test.java.srcDirs += 'src/test/java'
+    }
+    testOptions {
+        unitTests.includeAndroidResources = true
+        unitTests.all {
+            maxHeapSize = "1024m"
+        }
+    }
+    compileOptions {
+        sourceCompatibility JavaVersion.VERSION_1_8
```

Comment:
Could these be defined as a variable [here](https://github.com/RevenueCat/purchases-android/blob/main/build.gradle#L2_L6)? Looks like it's currently repeated in several places.

Reply:
Good idea. We can create a base .gradle for `api-tester`, `integration-tests`, `purchase-tester` and `purchase-tester-java`!

Reply:
Going to merge this and open another PR for that change